frontend/src/page/Realm.tsx

```diff
@@ -1,20 +1,49 @@
 import { jsx } from "@emotion/core";
 import React from "react";
-import { match } from "react-router-dom";
+import { Link, match } from "react-router-dom";
+
+import { Breadcrumbs } from "../ui/Breadcrumbs";
 
 
 type Props = {
     match: match<{ path?: string }>,
 };
 
 export const Realm: React.FC<Props> = ({ match }) => {
-    // const { path } = useParams();
-    const path = (match.params.path || "").split("/");
+    const isRoot = !match.params.path;
+    const currentPath = match.params.path || "";
```

Comment:
Why is `path` optional in the first place?

Reply:
See above, home page